### DISCUSSION TITLE
rheoFoam solver

### DISCUSSION THREAD

**Discussion by sunil-27** [fppimenta/rheoTool#83] [General]
While running basic case channel flow, using the RheoFoam, I'm facing the following issue 
**rheoFoam: error while loading shared libraries: libHYPRE-2.18.2.so: cannot open shared object file: No such file or directory**.
Can you please help me out with this?

**Comment by fppimenta** [comment]
The error tells you that  libHYPRE-2.18.2.so is not in your LD_LIBRARY_PATH (the .so either doesn't exist at all or it could just not be found). Several reasons could be behind, e.g. a bad installation or an incorrect definition of LD_LIBRARY_PATH in your .bashrc.

**Reply by sunil-27** [reply]
Thanks, it's working by appending source with lib path.

**Reply by sunil-27** [reply]
I'm trying to simulate fluid flow in idealized stenosis axisymmetric geometry, I have considered **sPTT constitutive equation** to model fluid and pulsatile inlet condition with time period T using **rheoFoam solver.** At the time step (T/4) vortices formation occurs at the post stenosis region, with an immediate effect there is a surge in courant no. I have done multiple trials to get rid of the rapid rise but nothing had worked in lowering them. Could you please help me in this regard?
